tests/MSBuildDeviceIntegration/Tests/BundleToolNoAbiSplitTests.cs

```diff
@@ -0,0 +1,153 @@
+
+			return ret;
+		}
+
+		[OneTimeSetUp]
+		public void OneTimeSetUp ()
+		{
+			var path = Path.Combine ("temp", TestName);
+
+			app = new XamarinFormsMapsApplicationProject {
+				IsRelease = true,
+				AotAssemblies = false, // Release defaults to Profiled AOT for .NET 6
+				PackageName = "com.xamarin.bundletoolnoabisplittests",
+			};
+			app.OtherBuildItems.Add (
+				MakeAndroidAsset (
+					include: "Assets\\Textures\\asset1.txt",
+					content: "Asset1"
+				)
+			);
+			app.OtherBuildItems.Add (
+				MakeAndroidAsset (
+					include: "Assets\\Textures#tcf_astc\\asset3.txt",
+					content: "Asset3astc",
+					assetPack: "assetpack1",
+					deliveryType: "InstallTime"
+				)
+			);
+			app.OtherBuildItems.Add (
+				MakeAndroidAsset (
+					include: "Assets\\Textures#tcf_paletted\\asset3.txt",
+					content: "Asset3",
+					assetPack: "assetpack1"
+				)
+			);
+			app.OtherBuildItems.Add (
+				MakeAndroidAsset (
+					include: "Assets\\Textures#tcf_etc1\\asset3.txt",
+					content: "Asset3etc1",
+					assetPack: "assetpack1"
+				)
+			);
+			app.OtherBuildItems.Add (
+				MakeAndroidAsset (
+					include: "Assets\\Textures#tcf_dxt1\\asset3.txt",
+					content: "Asset3Dx1",
+					assetPack: "assetpack1"
+				)
+			);
+			app.OtherBuildItems.Add (new BuildItem ("None", "buildConfig.config") {
+				TextContent = () => BuildConfig,
+			});
+
+			//NOTE: this is here to enable adb shell run-as
+			app.AndroidManifest = app.AndroidManifest.Replace ("<application ", "<application android:debuggable=\"true\" ");
+			app.SetProperty (app.ReleaseProperties, "AndroidPackageFormat", "aab");
+			app.SetRuntimeIdentifiers (RIDs);
+			app.SetProperty ("AndroidBundleConfigurationFile", "buildConfig.config");
+			app.SetProperty ("_FastDeploymentDiagnosticLogging", "true");
+
+			appBuilder = CreateApkBuilder (Path.Combine (path, app.ProjectName), cleanupOnDispose: true);
+			Assert.IsTrue (appBuilder.Build (app), "App build should have succeeded.");
+
+			var projectDir = Path.Combine (Root, appBuilder.ProjectDirectory);
+			intermediate = Path.Combine (projectDir, app.IntermediateOutputPath);
+			bin = Path.Combine (projectDir, app.OutputPath);
+
+			string objPath = Path.Combine (Root, appBuilder.ProjectDirectory, app.IntermediateOutputPath);
+                        List<EnvironmentHelper.EnvironmentFile> envFiles = EnvironmentHelper.GatherEnvironmentFiles (
+                                objPath,
+                                String.Join (";", RIDs.Select (rid => MonoAndroidHelper.RidToAbi (rid))),
+                                true
+                        );
+                        EnvironmentHelper.ApplicationConfig app_config = EnvironmentHelper.ReadApplicationConfig (envFiles);
+
+			Assert.That (app_config, Is.Not.Null, "application_config must be present in the environment files");
+                        Assert.AreEqual (app_config.ignore_split_configs, true, $"App config should indicate that split configs must be ignored");
```

Comment:
We should probably add additional code like

```
Assert.IsTrue (appBuilder.Install (app), $"{app.ProjectName} should install");
				RunProjectAndAssert (app, appBuilder);
				Assert.True (WaitForActivityToStart (app.PackageName, "MainActivity",
					Path.Combine (Root, appBuilder.ProjectDirectory, "logcat.log"), 30), "Activity should have started.");
```

To make sure the app actually installs and runs.

Reply:
I put the run code in a separate test [`InstallAndRun`](https://github.com/xamarin/xamarin-android/pull/8987/files#diff-95a43f2705e9ed1f4d6dc96b13bb7ab6245ad899c1a26073862c20bfb0ae3425R147-R151)